source/microbit/help.c

```diff
@@ -81,58 +81,65 @@ STATIC const mp_doc_t help_table_types[] = {
-    {&microbit_p16_obj, "Represents pin 16 on the gold edge connector.\n"},
-    {&microbit_p19_obj, "Represents pin 19 on the gold edge connector.\n"},
-    {&microbit_p20_obj, "Represents pin 20 on the gold edge connector.\n"},
-    {&microbit_pin_write_digital_obj, "write_digital(v) -- write a digital value v to the pin. Use 0 (lo) or 1 (hi).\n"},
-    {&microbit_pin_read_digital_obj, "read_digital() -- read the current digital value from the pin: 0 (lo) or 1 (hi).\n"},
-    {&microbit_pin_write_analog_obj, "write_analog(v) -- write an analog value v between 0..255 to the pin.\n"},
-    {&microbit_pin_read_analog_obj, "read_analog() -- read the current analog value from the pin: 0 - 65535.\n"},
-    {&microbit_pin_is_touched_obj, "is_touched() -- True if the pin is touched, otherwise False.\n"},
+    {&microbit_module, "Useful stuff to control the micro:bit hardware.\n"},
+    // System state objects
+    {&microbit_panic_obj, "Begin panic() mode (requires a reset).\n"},
+    {&microbit_random_obj, "Return a random(number) between 0 and 'number - 1'.\nrandom(10) returns a number between 0 and 9.\n"},
+    {&microbit_sleep_obj, "Put micro:bit to sleep(time) for some milliseconds (1 second = 1000 ms) of time.\nsleep(2000) gives micro:bit a 2 second nap.\n"},
+    {&microbit_running_time_obj, "Return running_time() in milliseconds since micro:bit's last reset.\n"},
+    // Accelerometer 3D orientation
+    {&microbit_accelerometer_obj, "Detect micro:bit's position (3D orientation).\n"},
+    {&microbit_accelerometer_get_x_obj, "Return micro:bit's X axis position.\n"},
+    {&microbit_accelerometer_get_y_obj, "Return micro:bit's Y axis position.\n"},
+    {&microbit_accelerometer_get_z_obj, "Return micro:bit's Z axis position.\n"},
+    // Pushbutton
+    {&microbit_button_a_obj, "micro:bit's 'A' button. When button is pressed down, is_pressed() is True.\n"},
+    {&microbit_button_b_obj, "micro:bit's 'B' button. When button is pressed down, is_pressed() is True.\n"},
+    {&microbit_button_is_pressed_obj, "If the button is pressed down, is_pressed() is True, else False.\n"},
+    // Compass 3D direction heading
+    {&microbit_compass_obj, "Use the compass to see what direction micro:bit is facing.\n"},
+    {&microbit_compass_is_calibrated_obj, "If micro:bit's compass is_calibrated() and adjusted for accuracy, return True.\nIf the compass hasn't been adusted for accuracy, return False.\n"},
+    {&microbit_compass_calibrate_obj, "Micro:bit, I command you to calibrate() and adjust your accuracy.\nWhere are my manners, please calibrate()"},
+    {&microbit_compass_is_calibrating_obj, "If micro:bit's compass is_calibrating() its accuracy, it sends True.\nIf it's not busy calibrating things, it sends False.\n"},
+    {&microbit_compass_clear_calibration_obj, "Reset micro:bit's compass using clear_calibration() command. Run calibrate() to improve accuracy.\n"},
+    {&microbit_compass_get_x_obj, "Return the compass' X axis direction.\n"},
```

Comment:
Whichever word is used, it'd be useful to have short description of what it represents, as most people would expect a heading from a compass rather than an x, y, z value.